04-Multi-Band-Raster.Rmd

```diff
@@ -117,104 +126,92 @@ par(original_par) # go back to original par
-collected using the NEON Airborne Observation Platform high resolution camera over
+collected using the <a href="http://http://www.neoninc.org/science-design/collection-methods/airborne-remote-sensing" target="_blank">NEON Airborne Observation Platform</a> high resolution camera over
 the <a href="http://www.neoninc.org/science-design/field-sites/harvard-forest" target="_blank">NEON Harvard Forest field site</a>. 
+These RGB images are 3-band rasters. Most of what we will do can be transferred 
+to rasters with a different number of bands.
 
-
-#Getting Started with Multi-Band data
+#Getting Started with Multi-Band Data
 If we read a raster stack into R using the `raster` function, it defaults to 
-reading in one (the first) band. We can plot this band using the plot function.
-
-In a typical GIS application, a single band would render a single image 
-band using a grayscale color palette. We will thus use a grayscale palette to render
-individual bands below.
-{: .notice }
+reading in just the first band. We can plot this band using the plot function. 
 
 ```{r read-single-band }
  
-# Read in multi-band raster with raster function, the default is the first band
+# Read in multi-band raster with raster function. 
+#Default is the first band only.
 RGB_band1 <- raster("NEON_RemoteSensing/HARV/HARV_RGB_Ortho.tif")
 
-#create a grayscale color palette to use for the image
-grayscale_colors <- gray.colors(100, 
-                                start = 0.0, 
-                                end = 1.0, 
-                                gamma = 2.2, 
-                                alpha = NULL)
+#create a grayscale color palette to use for the image.
+grayscale_colors <- gray.colors(100,            #how many different color levels 
+                                start = 0.0,    #how black (0) to go
+                                end = 1.0,      #how white (1) to go
+                                gamma = 2.2,    #correction between how a digital
+                                #camera sees the world and how human eyes see it
+                                alpha = NULL)   #Null=colors are not transparent
 
-#Point out dimension, CRS, and values attributes, but esp. band
+#Plot band 1
 plot(RGB_band1, 
-     col=grayscale_colors(100), 
+     col=grayscale_colors, 
      main="NEON RGB Imagery - Band 1\nHarvard Forest") 
 
-#view attributes
+#view attributes: Check out dimension, CRS, resolution, values attributes, and 
+#band.
 RGB_band1
-
-
 ```
 
 Note that when we look at the attributes of RGB_Band1, we see :
 
 `band: 1  (of  3  bands)`
 
-This is R telling us that this particular raster object has more bands associated with it!
+This is `R` telling us that this particular raster object has more bands
+associated with it!
 
-##Raster Data Values - Images
-
-Take careful note of the min and max values of our raster min and max values:
+>Bonus: The number of bands associated with a raster object can also be
```

Comment:
Moved from challenge later to bonus here since we were discussing how to view # of bands.  Again, slots appearing (also in vector) although in both instances I think participant can understand the concept without understanding slots.